### ISSUE TITLE
Acrylic Paint does not apply a funny "approved by Richard Stallman" overlay over artworks created before saying "GNU plus Linux" in the microphone

### ISSUE THREAD

**Issue by valerio-bozzolan** [valerio-bozzolan/AcrylicPaint#40]
This funny title is a.k.a.:
# Questions before publishing every new enhancement in the bug report of the `$xyz` project

### Yes
People will discuss your implementation! Someone will read your work and will say: «*Ehy wait, I'm already working on it! Let's merge our work!*» or another will start a discussion etc.

Publish that new enhancement as a bug report: your pull request will be very appreciated!

### No
Bug reports are a **discussion** area and **not a shopping list**! The project is not so active? Nobody contribute? Instead of writing down what the software does and what the software does not (because of we **know e-x-a-c-t-l-y** what the software does and what the software does not) please **don't ask for enhancements in the bug report**! Instead, in order of preference:
* **Get that damn source code and fix it yourself! The community will appreciate your contributions**!!!
* Hire the maintainer, a contributor o a friend in order to do your job for you! (**it's normal to pay someone to work for you**.)

People behind a Free as in Freedom project usually have a life: they study, work, love, pay taxes, etc. and so they are not heroes, they don't work in the name of the public interest and so **they don't work for you!** So avoid this sort of funny messages:

    This project is great! I use it everyday! But please add this feature! I can't implement it by myself. I love you! <3 <3 <3

TL;DR
If you can do it: **do it** or at least **try it**!
If you can't try to do it: donate or hire someone!

**Comment by valerio-bozzolan** [comment]
For the records: don't take this too seriously! BTW...

Less words, more hack! :smile:

**Comment by valerio-bozzolan** [comment]
This seems related to https://github.com/mishoo/UglifyJS2/issues/2054 somehow. :^)
